crawler-cli/src/main/java/org/gbif/crawler/dwca/avroconverter/DwCAToAvroService.java

```diff
@@ -0,0 +1,35 @@
+package org.gbif.crawler.dwca.avroconverter;
+
+import org.gbif.common.messaging.MessageListener;
+
+import com.google.common.util.concurrent.AbstractIdleService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A service which listens to the  {@link org.gbif.common.messaging.api.messages.DwcaValidationFinishedMessage } and perform conversion
+ */
+public class DwCAToAvroService extends AbstractIdleService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(DwCAToAvroService.class);
+  private final DwCAToAvroConfiguration configuration;
+  private MessageListener listener;
+
+  public DwCAToAvroService(DwCAToAvroConfiguration configuration) {
+    this.configuration = configuration;
+  }
+
+  @Override
+  protected void startUp() throws Exception {
+    LOG.info("Started dwca-to-avro service with parameters : " + configuration.toString());
```

Comment:
use something like LOG.info("Started dwca-to-avro service with parameters : {} ", configuration)); instead

Reply:
done